index.js

```diff
@@ -62,22 +76,27 @@ conventionalRecommendedBump({
     console.error(chalk.red(err.message))
     return
   }
-
   var newVersion = pkg.version
   if (!argv.firstRelease) {
-    newVersion = semver.inc(pkg.version, release.releaseAs)
+    newVersion = getNextVersion(release)
     checkpoint('bumping version in package.json from %s to %s', [pkg.version, newVersion])
     pkg.version = newVersion
     fs.writeFileSync(pkgPath, JSON.stringify(pkg, null, 2) + '\n', 'utf-8')
   } else {
     checkpoint('skip version bump on first release', [], chalk.red(figures.cross))
   }
-
-  outputChangelog(argv, function () {
+  // only commit the package.json on prerelease
```

Comment:
So the version for a prerelease is bumped, but no docs are generated. This only happens when the release is stable. It would flood the changelog otherwise.